code/modules/asset_cache/asset_cache_client.dm

```diff
@@ -36,7 +36,7 @@
 	var/job = ++last_asset_job
 	var/t = 0
 	var/timeout_time = timeout
-	src << browse({"<script>window.location.href="?asset_cache_confirm_arrival=[job]"</script>"}, "window=asset_cache_browser&file=asset_cache_send_verify.htm")
+	src << browse({"<script>window.location.href="byond://?asset_cache_confirm_arrival=[job]"</script>"}, "window=asset_cache_browser&file=asset_cache_send_verify.htm")
```

Comment:
This don't quite look like the others. Did someone just blindly hit replace-all for the whole codebase?